modules/meeting/app/services/meetings/icalendar_builder.rb

```diff
@@ -190,6 +198,7 @@ def series_cache_loaded?
     def build_icalendar
       ::Icalendar::Calendar.new.tap do |calendar|
         calendar.prodid = "-//OpenProject GmbH//#{OpenProject::VERSION}//Meeting//EN"
+        calendar.refresh_interval = "PT6H"
```

Comment:
:green_circle: What do you think about using active support to help building this string? That might be slightly more readable for anyone that doesn't immediately know what an ISO8601 duration looks like:

```suggestion
        calendar.refresh_interval = 6.hours.iso8601
```

Reply:
TIL that active support has that built in 👍